column_mapping.go

```diff
@@ -40,7 +41,8 @@ func (m *ColumnMapping) String() string {
 
 	if len(m.columns) > 0 {
 		for _, path := range m.columns {
-			fmt.Fprintf(s, "\n  % 2d => %q", m.ColumnIndex(path...), columnPath(path))
+			columnIndex, _ := m.Lookup(path...)
+			fmt.Fprintf(s, "\n  % 2d => %q", columnIndex, columnPath(path))
```

Comment:
Just a guess that most pages will have 99 columns or fewer?

Reply:
I think I was more concerned that the visual impact will be felt when displaying 10-20 columns, beyond hundreds this format is likely the wrong tool and if the alignment is inaccurate it's not the worst part of the experience anyways.